contrib/PKGBUILD

```diff
@@ -1,45 +1,161 @@
-# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Maintainer: Bruno Pagani <archange@archlinux.org>
+# Maintainer: Filipe Laíns (FFY00) <lains@archlinux.org>
+# Maintainer: Frederik Schwan <freswa at archlinux dot org>
 # Contributor: Mirco Tischler <mt-ml at gmx dot de>
 
-pkgname=fwupd
-pkgver=dummy
-pkgrel=1
-pkgdesc='A system daemon to allow session software to update firmware'
-arch=('i686' 'x86_64')
+pkgbase=fwupd
+pkgname=(
+  fwupd
+  fwupd-docs
+)
+pkgver=1.9.18
+pkgrel=2
```

Comment:
I guess we'll probably want to make these "dummy" again or auto populated in some way?